### PR TITLE
Correct style using rubocop (part 1)

### PR DESCRIPTION
**Pull request by astrozzc**
Used cop:
Style/TrivialAccessors, Style/NumericPredicate

@miq-bot add_label ivanchuk/no, refactoring
@miq-bot add_reviewer @gmcculloug 
@miq-bot assign @tinaafitz

### CODE REVIEW

**Comment by astrozzc** [comment]
@miq-bot assign @tinaafitz

File: spec/support/miq_ae_mock_object.rb [hunk 1]
```diff
@@ -1,11 +1,8 @@
 module Spec
   module Support
     class MiqAeMockObject
-      attr_reader :parent
       attr_reader :children
-      attr_accessor :namespace
-      attr_accessor :klass
-      attr_accessor :instance
+      attr_accessor :namespace, :klass, :instance, :parent
```
Comment (gmcculloug) [inline spec/support/miq_ae_mock_object.rb]: Minor, but let's put these in alphabetical order.

**Comment by gmcculloug** [comment]
Let's address to two rubocop issues that came up as well.

File: spec/support/miq_ae_mock_object.rb (hunk 1 above)
Comment (gmcculloug) [inline spec/support/miq_ae_mock_object.rb]: @mkanoor This looks good to me, just looking for a second review.

File: spec/support/miq_ae_mock_service.rb [hunk 2]
```diff
@@ -10,27 +10,20 @@ class MiqAeMockService
       include MiqAeMethodService::MiqAeServiceObjectCommon
       include MiqAeMethodService::MiqAeServiceModelLegacy
 
-      attr_reader :root, :object
-      attr_accessor :inputs
+      attr_reader :root
+      attr_writer :current_object
+      attr_accessor :inputs, :object
```
Comment (gmcculloug) [inline spec/support/miq_ae_mock_service.rb]: @mkanoor Same here.

Comment (mkanoor) [reply spec/support/miq_ae_mock_service.rb]: 👍

File: spec/support/miq_ae_mock_object.rb (hunk 1 above)
Comment (mkanoor) [reply spec/support/miq_ae_mock_object.rb]: 👍

**Pull request merged**